npm-shrinkwrap.json

```diff
@@ -1858,6 +2232,820 @@
       "resolved": "http://registry.npmjs.org/chokidar/-/chokidar-1.6.1.tgz",
       "dev": true,
       "dependencies": {
+        "fsevents": {
```

Comment:
I've forgot to run `npm install` with `--no-optional` 😊 I've to admit that I never used yarn before. I'm happy to give it a try, finally.